website/blog/2024-03-18-deploying-offline-with-docker/index.md

```diff
@@ -0,0 +1,69 @@
+---
+authors: [wwayne]
+tags: [deployment]
+---
+
+# Deploying offline with Docker
+
+<div align="center">
+
+![No internet access](./no-internet.png)
```

Comment:
Make the image smaller / more propotional?

Reply:
Updated, attached the screenshot in the comment below